inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_broadcast.cpp

```diff
@@ -29,19 +31,66 @@ void dynamicToStaticShapeBroadcast(std::shared_ptr<ngraph::Node> target) {
                 broadcast->input_value(0),
                 broadcast->input_value(1),
                 broadcast->input_value(2));
-    } else if (broadcast->get_broadcast_spec() == ngraph::op::BroadcastType::NUMPY) {
+    } else if (broadcast->get_broadcast_spec() == ngraph::op::BroadcastType::NUMPY ||
+               broadcast->get_broadcast_spec() == ngraph::op::BroadcastType::BIDIRECTIONAL) {
         staticShapeBroadcast = std::make_shared<ngraph::vpu::op::StaticShapeBroadcast>(
                 broadcast->input_value(0),
-                broadcast->input_value(1));
+                broadcast->input_value(1),
+                broadcast->get_broadcast_spec());
     } else {
-        VPU_THROW_FORMAT("dynamicToStaticShapeBroadcast supports only explicit and numpy modes,"
+        VPU_THROW_FORMAT("dynamicToStaticShapeBroadcast supports only explicit, numpy and bidirectional modes,"
                          "provided {}", broadcast->get_broadcast_spec().m_type);
     }
 
-    auto dsr = std::make_shared<ngraph::vpu::op::DynamicShapeResolver>(
-            staticShapeBroadcast->output(0), broadcast->input_value(1));
-    dsr->set_friendly_name(broadcast->get_friendly_name());
+    std::shared_ptr<ngraph::Node> dsr;
+
+    if (broadcast->get_broadcast_spec() == ngraph::op::BroadcastType::BIDIRECTIONAL) {
+        const auto inputShape = broadcast->get_input_partial_shape(0);
```

Comment:
Nitpick: you can get static shape by `get_input_shape(0)`

Reply:
Fixed.